pkg/sdn/plugin/node.go

```diff
@@ -63,6 +78,10 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient *kclien
 		}
 	}
 
+	if err := writeCNIConfig(masterKubeConfig, hostname, mtu); err != nil {
```

Comment:
These are all runtime options though, and I'd rather pull them from the same source than have to write the same information in two different places at install time, one of which is highly likely to be forgotten if anyone updates the openshift config file.